sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/core/properties/retry/HttpRetryProperties.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.core.properties.retry;
+
+import java.time.temporal.ChronoUnit;
+
+/**
+ * Unified http retry properties for all Azure SDKs based on HTTP.
+ */
+public class HttpRetryProperties extends RetryProperties {
+
+    private String retryAfterHeader;
+    private ChronoUnit retryAfterTimeUnit;
```

Comment:
ChronoUnit why are we using this

Reply:
It used to create com.azure.core.http.policy.RetryPolicy#RetryPolicy(com.azure.core.http.policy.RetryStrategy, java.lang.String, java.time.temporal.ChronoUnit).